.github/workflows/ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: Tests in flink-kubernetes-operator
         run: |
           cd flink-kubernetes-operator
-          mvn integration-test -Dit.skip=false
+          mvn verify -Dit.skip=false
```

Comment:
I am afraid the `org.apache.flink.kubernetes.operator.FlinkOperatorITCase` is not triggered by using `mvn verify`. Could you please confirm this?

Reply:
Could you please share a failed CI when using `mvn verify`?

Reply:
Sure, you can see failed CI [here](https://github.com/bgeng777/flink-kubernetes-operator/runs/6089734117?check_suite_focus=true#step:10:2083)